app-modules/ai/src/Jobs/CloneAiThread.php

```diff
@@ -67,8 +67,9 @@ public function middleware(): array
 
     public function handle(): void
     {
-        $threadReplica = $this->thread->replicate(except: ['id', 'thread_id', 'folder_id']);
+        $threadReplica = $this->thread->replicate(except: ['id', 'thread_id', 'folder_id', 'saved_at', 'emailed', 'cloned']);
```

Comment:
You missed renaming emailed and cloned here.

Reply:
Sorry, I forgot in hurry. resolved now.